BankSDK/GiniBankSDKExample/Tests/TransferSummaryIntegrationTest.swift

```diff
@@ -0,0 +1,221 @@
+     */
+    class CaptureResultsDelegateForTransferSummaryTest: GiniCaptureResultsDelegate {
+
+        let integrationTest: TransferSummaryIntegrationTest
+        let expect: XCTestExpectation
+
+        init(integrationTest: TransferSummaryIntegrationTest, expect: XCTestExpectation) {
+            self.integrationTest = integrationTest
+            self.expect = expect
+        }
+
+        func giniCaptureAnalysisDidFinishWith(result: AnalysisResult) {
+            // 1b. Received the extractions for the uploaded document
+            let fixtureExtractionsJson = self.integrationTest.loadFile(withName: "result_Gini_invoice_example_payment_reference", ofType: "json")
+
+            let fixtureExtractionsContainer = try! JSONDecoder().decode(ExtractionsContainer.self, from: fixtureExtractionsJson)
+
+            // 2. Verify we received the correct extractions for this test
+            XCTAssertEqual(fixtureExtractionsContainer.extractions.first(where: { $0.name == "iban" })?.value,
+                           result.extractions["iban"]?.value)
+            XCTAssertEqual(fixtureExtractionsContainer.extractions.first(where: { $0.name == "paymentRecipient" })?.value,
+                           result.extractions["paymentRecipient"]?.value)
+//            XCTAssertEqual(fixtureExtractionsContainer.extractions.first(where: { $0.name == "paymentPurpose" })?.value,
+//                           result.extractions["paymentPurpose"]?.value)
+            XCTAssertEqual(fixtureExtractionsContainer.extractions.first(where: { $0.name == "bic" })?.value,
+                           result.extractions["bic"]?.value)
+            XCTAssertEqual(fixtureExtractionsContainer.extractions.first(where: { $0.name == "amountToPay" })?.value,
+                           result.extractions["amountToPay"]?.value)
+
+            // 3. Assuming the user saw the following extractions:
+            //    amountToPay, iban, bic, paymentPurpose and paymentRecipient
+            //    Supposing the user changed the amountToPay from "995.00:EUR" to "950.00:EUR"
+            //    we need to update that extraction
+            result.extractions["amountToPay"]?.value = "950.00:EUR"
+
+            if result.extractions["amountToPay"] != nil {
+
+                DispatchQueue.main.asyncAfter(deadline: .now() + 10, execute: {
+                    // 5. Verify that the extractions were updated
+                    self.integrationTest.getUpdatedExtractionsFromGiniBankSDK(for: result.document!) { result in
+                        switch result {
+                        case let .success(extractionResult):
+                            let extractionsAfterFeedback = extractionResult.extractions
+
+
+                            let fixtureExtractionsAfterFeedbackJson = self.integrationTest.loadFile(withName: "result_Gini_invoice_example_payment_reference_after_feedback", ofType: "json")
```

Comment:
[clarification]: do we want to rename the file accordingly from `result_Gini_invoice_example_payment_reference_after_feedback` to `result_Gini_invoice_example_payment_reference_after_transfer_summary`?
Also the properties: `extractionsAfterFeedback`, `fixtureExtractionsAfterFeedbackJson`, `fixtureExtractionsAfterFeedbackContainer`?

Reply:
We keep the current names